### PR TITLE
ENG-1163 Implement OOTB page templates in new Entando install

### PR DESCRIPTION
**Pull request by ichalagashvili**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **joewhite101**

If they are not already assigned, you can assign the PR to them by writing `/assign @joewhite101` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-de-app).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando-k8s/entando-de-app/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/main/resources/sql/defaultResources/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/defaultResources/port_data_production.sql]:
```suggestion
      <descr>Frame 1</descr>
```

File: src/main/resources/sql/defaultResources/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/defaultResources/port_data_production.sql]:
```suggestion
      <descr>Frame 2</descr>
```

File: src/main/resources/sql/defaultResources/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/defaultResources/port_data_production.sql]:
```suggestion
      <descr>Frame 3</descr>
```

File: src/main/resources/sql/defaultResources/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/defaultResources/port_data_production.sql]:
```suggestion
      <descr>Frame 4</descr>
```

File: src/main/resources/sql/defaultResources/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/defaultResources/port_data_production.sql]:
```suggestion
      <descr>"Frame 5</descr>
```

File: src/main/resources/sql/defaultResources/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/defaultResources/port_data_production.sql]:
```suggestion
      <descr>Frame 6</descr>
```

File: src/main/resources/sql/defaultResources/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/defaultResources/port_data_production.sql]:
```suggestion
      <descr>Frame 7</descr>
```

File: src/main/resources/sql/defaultResources/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/defaultResources/port_data_production.sql]:
```suggestion
      <descr>Frame 8</descr>
```

File: src/main/resources/sql/defaultResources/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/defaultResources/port_data_production.sql]: please add the css locally

File: src/main/resources/sql/defaultResources/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/defaultResources/port_data_production.sql]:
```suggestion
          <div class="Homepage4 Homepage__header"><@wp.show frame=0 /></div>
```

File: src/main/resources/sql/defaultResources/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/defaultResources/port_data_production.sql]:
```suggestion
          <div class="Homepage3 Homepage__header"><@wp.show frame=0 /></div>
```

File: src/main/resources/sql/defaultResources/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/defaultResources/port_data_production.sql]:
```suggestion
          <div class="Homepage2 Homepage__header"><@wp.show frame=0 /></div>
```

File: src/main/resources/sql/defaultResources/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/defaultResources/port_data_production.sql]:
```suggestion
      <descr>Frame 1</descr>
```

File: src/main/resources/sql/defaultResources/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/defaultResources/port_data_production.sql]:
```suggestion
      <descr>Frame 2</descr>
```

File: src/main/resources/sql/defaultResources/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/defaultResources/port_data_production.sql]:
```suggestion
      <descr>Frame 3</descr>
```

File: src/main/resources/sql/defaultResources/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/defaultResources/port_data_production.sql]:
```suggestion
      <descr>Frame 4</descr>
```

File: src/main/resources/sql/defaultResources/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/defaultResources/port_data_production.sql]:
```suggestion
      <descr>"Frame 5</descr>
```

File: src/main/resources/sql/defaultResources/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/defaultResources/port_data_production.sql]:
```suggestion
      <descr>Frame 6</descr>
```

File: src/main/resources/sql/defaultResources/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/defaultResources/port_data_production.sql]:
```suggestion
      <descr>Frame 7</descr>
```

File: src/main/resources/sql/defaultResources/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/defaultResources/port_data_production.sql]:
```suggestion
      <descr>Frame 8</descr>
```

File: src/main/resources/sql/defaultResources/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/defaultResources/port_data_production.sql]:
```suggestion
      <descr>Frame 9</descr>
```

File: src/main/resources/sql/defaultResources/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/defaultResources/port_data_production.sql]:
```suggestion
      <descr>Frame 10</descr>
```

File: src/main/resources/sql/defaultResources/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/defaultResources/port_data_production.sql]: please add the css locally

File: src/main/resources/sql/defaultResources/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/defaultResources/port_data_production.sql]: please add the css locally

File: src/main/webapp/resources/static/css/ootb/page-templates/index.css
Comment (gidesan) [inline src/main/webapp/resources/static/css/ootb/page-templates/index.css]: For this one and other `HomepageX__header` classes, try to extract the common part to a `Homepage__header` class, then use the specific parts using `.HomepageX.Homepage__header` selector.

Also ensure all is working fine, because the HTML templates only had `HomepageX_header` (with only one underscore char instead of two).

**Review by gidesan — CHANGES_REQUESTED** [review]

File: src/main/resources/sql/defaultResources/port_data_production.sql
Comment (ichalagashvili) [reply src/main/resources/sql/defaultResources/port_data_production.sql:1030]: yes

**Review by gidesan — APPROVED** [review]
well done! :)

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando-k8s/entando-de-app/pull/80#pullrequestreview-489542367" title="Approved">gidesan</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **joewhite101**

If they are not already assigned, you can assign the PR to them by writing `/assign @joewhite101` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-de-app).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando-k8s/entando-de-app/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-jx** [comment]
@ichalagashvili: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 65bba63bf766e6ae9037e414dacf6dfb77a632df | [link](https://raw.githubusercontent.com/entando-k8s/devops-results/logs/jenkins-x/logs/entando-k8s/entando-de-app/PR-80/2.log) | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Pull request merged**
